fabric8-maven-plugin-build/src/test/java/org/fabric8/maven/plugin/build/ResourceGeneratorBuilderIT.java

```diff
@@ -0,0 +1,58 @@
+package org.fabric8.maven.plugin.build;
+
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.TemporaryFolder;
+
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(RequiresOpenshift.class)
+@RequiresOpenshift
+public class ResourceGeneratorBuilderIT {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void should_generate_build_and_resources() throws IOException {
```

Comment:
Minor detail: I'm not sure if we generate the build per se. Might be a bit misleading name later.